build_library/vm_image_util.sh

```diff
@@ -883,10 +883,12 @@ _write_qemu_uefi_secure_conf() {
             ;;
     esac
 
+    # TODO: Remove the temporary flatcar shim signing cert
```

Comment:
Is this `/usr/share/sb_keys/DB.crt` the temporary flatcar shim signing cert? What are the plans with its removal?

Reply:
The temporary flatcar shim signing cert is the newly added `flatcar-sb-dev-shim-2025.cert` itself. Currently, we are working to submit our shim for submission, it's been currently been worked on with CNCF. Once the shim submission is done to shim-review, and then Microsoft signs our shim - we will remove this temporary cert. 

The `/usr/share/sb_keys/DB.crt` is used for the developer builds.

Reply:
Gotcha, thanks!